lib/bindings/MQTTBinding.js

```diff
@@ -272,15 +352,19 @@ function start(newConfig, callback) {
 }
 
 function stop(callback) {
-    logger.info('Stopping IoT Agent');
+    logger.info('Stopping MQTT Binding' + (new Date()));
 
     async.series([
         unsubscribeAll,
         mqttClient.end.bind(mqttClient, true)
-    ], callback);
+    ], function() {
+        logger.info('MQTT Binding Stopped' + (new Date()));
```

Comment:
Is it really needed to log the current date and time since already included in the `time` property of the message logged by `logops`?